src/bento/components/bento-autocomplete/1.0/test/test-bento-autocomplete.js

```diff
@@ -0,0 +1,157 @@
+import {expect} from 'chai';
+
+import {BaseElement as BentoAutocomplete} from '#bento/components/bento-autocomplete/1.0/base-element';
+
+import {htmlFor} from '#core/dom/static-template';
+
+import {defineBentoElement} from '#preact/bento-ce';
+
+import {user} from '#utils/log';
+
+import {waitFor} from '#testing/helpers/service';
+
+describes.realWin('bento-autocomplete:1.0', {amp: false}, (env) => {
```

Comment:
These tests only pass with the `--bento-only` flag, and that flag isn't present in CI. Do we need to fix something here in order to work with bento components that don't render in amp mode?

Reply:
This is not documented (I'll add it!), but you have to place this under `standalone/`
```diff
- src/bento/components/bento-autocomplete/1.0/test/test-bento-autocomplete.js
+ src/bento/components/bento-autocomplete/1.0/test/standalone/test-bento-autocomplete.js
```

Reply:
I did this, and I'm still running into the same issue. I'll push it up and try in CI.

Reply:
It seems to be passing in CI now 👍